execution/gethexec/sequencer.go

```diff
@@ -356,17 +357,33 @@ func NewSequencer(execEngine *ExecutionEngine, l1Reader *headerreader.HeaderRead
 		}
 		senderWhitelist[common.HexToAddress(address)] = struct{}{}
 	}
+
+	var (
+		lightClientReader *lightclient.LightClientReader
+		err               error
+	)
+
+	if l1Reader != nil {
```

Comment:
we should add a comment here describing why we are creating a light client reader here for other reader

Reply:
Agreed :+1: